frontend/web/routes.js

```diff
@@ -68,6 +69,7 @@ export const routes = {
   'environment-settings':
     '/project/:projectId/environment/:environmentId/settings',
   'sdk-keys': '/project/:projectId/environment/:environmentId/sdk-keys',
+  'identities': '/project/:projectId/environment/:environmentId/sdk-identities',
```

Comment:
The url here is a bit confusing, the existing users page could also be described as "sdk identities"

Reply:
Changed url to identities-stream, we feel this would be less confusing.